src/Microsoft.Android.Build.BaseTasks/Files.cs

```diff
@@ -28,6 +38,33 @@ public static class Files
 		public static readonly Encoding UTF8withoutBOM = new UTF8Encoding (encoderShouldEmitUTF8Identifier: false);
 		readonly static byte[] Utf8Preamble = Encoding.UTF8.GetPreamble ();
 
+		/// <summary>
+		/// Checks for the environment variable DOTNET_ANDROID_FILE_WRITE_RETRY_ATTEMPTS to
+		/// see if a custom value for the number of times to retry writing a file has been 
+		/// set.
+		/// </summary>
+		/// <returns>The value of DOTNET_ANDROID_FILE_WRITE_RETRY_ATTEMPTS or the default of DEFAULT_FILE_WRITE_RETRY_ATTEMPTS</returns>
+		public static int GetFileWriteRetryAttempts ()
+		{
+			var retryVariable = Environment.GetEnvironmentVariable ("DOTNET_ANDROID_FILE_WRITE_RETRY_ATTEMPTS");
+			if (!string.IsNullOrEmpty (retryVariable) && int.TryParse (retryVariable, out int retry))
```

Comment:
The value should be cached on first access.  Unless you have a unit test setting this variable, I don't see the need to have the performance overhead of checking + parsing this value all the time.